component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateAuthenticator.java

```diff
@@ -132,38 +132,33 @@ protected void processAuthenticationResponse(HttpServletRequest httpServletReque
                 Map<ClaimMapping, String> claims;
                 claims = getSubjectAttributes(authenticationContext, certAttributes);
                 AuthenticatedUser authenticatedUser = getUsername(authenticationContext);
-                String userName;
-                if (authenticatedUser != null) {
-                    if (log.isDebugEnabled()) {
-                        log.debug("Authenticated username is: " + authenticatedUser);
-                    }
-                    userName = authenticatedUser.getAuthenticatedSubjectIdentifier();
-                } else {
+                String userName = (String) authenticationContext.getProperty(X509CertificateConstants
```

Comment:
break the line after the operator.

Reply:
Fixed in 58315e20dcc5c17bcb4b10b4a40a542488d9324d